### DISCUSSION TITLE
论文的摘要、致谢部分

### DISCUSSION THREAD

**Discussion by Zhuzy0608** [note286/xduts#119] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/130884066/233382134-b03675d2-10dc-4f22-8202-9b5b4a33ad84.png)
设置完路径后，对应路径下的文档里面，书写格式有没有例子，或者是框架，（小白，不会弄这个，非常感谢学长的答疑）

**Comment by note286** [comment, marked as answer]
文件里直接写内容。
